KVKCalendar/Classes/EventPageView.swift

```diff
@@ -24,12 +24,12 @@ final class EventPageView: UIView {
         return text
     }()
     
-    private lazy var iconFileImageView: UIImageView = {
-        let image = UIImageView(frame: CGRect(x: 0, y: 2, width: 10, height: 10))
-        image.image = style.iconFile.withRenderingMode(.alwaysTemplate)
-        image.tintColor = style.colorIconFile
-        return image
-    }()
+//    private lazy var iconFileImageView: UIImageView = {
```

Comment:
Why do you comment this code?

Reply:
Hi i have comment this code because if you go to week view the app Crash.
is only for my test. you can skip this file